edb/edgeql/codegen.py

```diff
@@ -1628,25 +1628,25 @@ def visit_FuncParam(self, node: qlast.FuncParam) -> None:
             self.visit(node.default)
 
     def visit_ConfigSet(self, node: qlast.ConfigSet) -> None:
-        self.write('CONFIGURE')
-        self.write(' SYSTEM' if node.system else ' SESSION')
+        self.write('CONFIGURE ')
+        self.write(node.scope)
```

Comment:
Looks like it emits `DATABASE` rather than `CURRENT DATABASE`?

Reply:
Good catch.

Reply:
I see this comment marked as "resolved" but I don't see how it was fixed.

Reply:
In general I'd suggest adding an explicit `to_edgeql()` method to the enum, like other enums have.

Reply:
nvm, I see this was addressed in a follow up commit